src/calculateRentalCost.js

```diff
@@ -28,7 +28,20 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const costs = 40;
+  let result = days * costs;
+  const discountThreeDays = 20;
+  const discountSevenDays = 50;
```

Comment:
what if we will change these days numbers - we should rename variables then - so fix names in some way